docker/docker/src/main/java/org/arquillian/cube/docker/impl/model/DockerCube.java

```diff
@@ -139,10 +155,14 @@ public void stop() throws CubeControlException {
         }
         try {
             lifecycle.fire(new BeforeStop(id));
+
+            long currentTime = System.currentTimeMillis();
             try {
                 executor.stopContainer(id);
             } catch(NotFoundException e) {
             } catch (NotModifiedException e) {}
+            this.stoppingTimeInMillis = System.currentTimeMillis() - currentTime;
+
```

Comment:
I thought the same but notice that there are some before/after events. So if we took there it means that we could add to the time, the duration of the execution of events thrown there. So I decided that the most real time was doing in that way, although as you said it seems event would be better. WDYT?